frontend/src/editor/slate_types.d.ts

```diff
@@ -1,10 +1,11 @@
 import { BaseEditor } from 'slate';
 import { ReactEditor } from 'slate-react';
 import { Paragraph, Text } from './types';
+import { AutomergeEditor } from 'slate-automerge-doc';
 
 declare module 'slate' {
   interface CustomTypes {
-    Editor: BaseEditor & ReactEditor;
+    Editor: BaseEditor & ReactEditor & AutomergeEditor;
```

Comment:
`AutomergeEditor` is already `BaseEditor & ...`, so the following should work?

```suggestion
    Editor: AutomergeEditor & ReactEditor;
```